optuna/cli.py

```diff
@@ -58,8 +58,6 @@ def get_parser(self, prog_name: str) -> ArgumentParser:
         parser.add_argument(
             "--direction",
             type=str,
-            choices=("minimize", "maximize"),
```

Comment:
I dropped `choice` argument here because we can't check a value of `directions` (comma separated string) in argparse. Instead, `direction` and `directions` are validated in `take_action`.

https://github.com/optuna/optuna/pull/2640/files#diff-eed7817f413ac25f15f88a814790fde8684fd3d551916dec485cbda0c207845cR94-R95

Reply:
Thanks to the awesome suggestion, I restored `choices` parameter (and also introduced it to `--directions`).
https://github.com/optuna/optuna/pull/2640#discussion_r627035901